pkg/cloud/cloud.go

```diff
@@ -69,11 +69,15 @@ type FileSystemOptions struct {
 	CapacityGiB              int64
 	SubnetId                 string
 	SecurityGroupIds         []string
+	AutoImportPolicy         string
 	S3ImportPath             string
 	S3ExportPath             string
 	DeploymentType           string
 	KmsKeyId                 string
 	PerUnitStorageThroughput int64
+	DailyAutomaticBackupStartTime string
```

Comment:
Minor: em. This should be aligned. Seems this project doesn't have format check.

Reply:
fixed